### ISSUE TITLE
Records page - screen reader user improvement

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#464]
The table caption, heard by screen-reader users, is for example, "Information of Hackney District". To sound more natural, the "of" should be replaced with "about".

**Comment by richardofsussex** [comment]
This change has already been implemented (some time ago?!), so this issue can be closed.
